share/eds/pubsub.go

```diff
@@ -57,23 +57,10 @@ func (s *PubSub) Start(context.Context) error {
 // * Unregisters all the added Validators
 // * Closes the `ShrEx/Sub` topic
 func (s *PubSub) Stop(context.Context) error {
-	err := s.pubSub.UnregisterTopicValidator(s.pubSubTopic)
-	if err != nil {
-		return err
-	}
-
+	// TODO(@vgonkivs): unregister the topic validator
 	return s.topic.Close()
 }
 
-// AddValidator registers given Validator for EDS notifications (DataHash).
-// Any amount of Validators can be registered.
-func (s *PubSub) AddValidator(validate Validator) error {
```

Comment:
why? are you adding it back in another pr?

Reply:
yes. I do not want to keep this logic until the subscriber will be implemented.

Reply:
Ahhh, then why did you keep it in the original PR??? Pls return it back